tslib/src/functions/check-validator/index.ts

```diff
@@ -1,5 +1,5 @@
 import { AzureFunction, Context } from '@azure/functions'
-import { ValidateCheckMessageV1 } from '../../schemas'
+import { ValidateCheckMessageV1 } from '../../schemas/Models'

```

Comment:
Capitalised directory names are so Microsoft.  I'd prefer all lowercase, but can live with it.

Reply:
that's actually a file, called `Models.ts` and is inconsistent to other files in same project, will lower case them